controller/handler/trial.js

```diff
@@ -59,14 +60,16 @@ function trialView (request, reply) {
                 JOIN stage AS st
                 ON st.id = pa.stageId
                 WHERE st.trialId = ?
-                AND si.endTime > ?
+                AND si.startTime > ?
```

Comment:
Here should we be having the date comparisons as below :
si.startTime >= ?
si.endTime <= ?

@ChristianMurphy : Please comment and let us know.